alma/request.py

```diff
@@ -35,16 +47,19 @@ def set_query_params(self, params):
         self.params = params
         return self
 
+    @configure_credentials
     def get(self):
-        res = requests.get(self.url, self.params, headers=self.headers)
+        res = requests.get(self.url, self.params, headers=self.headers, cookies=self.cookies)
```

Comment:
I guess `_process_response` should initialize the cookies if not set not all the methods.
```suggestion
        res = requests.get(self.url, self.params, *, headers=None, cookies=None)
```